Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -1058,4 +1083,10 @@ double HoI4::Country::calculateInfluenceFactor()
 		//it's displayed ingame as being halfway: 0.5 instead of 0.0000something
 		return std::clamp(influenceFactor, 1.0, 100.0);
 	}
+}
+
+void HoI4::Country::updateConquerStrategy(std::string HoI4Tag, int valueToAdd)
+{
+	auto& theStrategy = conquerStrategies.find(HoI4Tag)->second;
```

Comment:
always always verify if find returned a valid reference by checking 
if (theStrategy == conquerStrategies.end()) throw runtime_exception("Shit!");

Reply:
It should be impossible to not find it since updateConquerStrategy is only called
`if (HoI4Tag)`